rdrf/rdrf/templates/registration/login_assistance_email.html

```diff
@@ -0,0 +1,18 @@
+{% load i18n %}{% autoescape off %}
+Hello  Mr/Ms {{user.last_name}},
+
+{% blocktrans %}You're receiving this email because you were having trouble signing in and requested help.{% endblocktrans %}
+
+{% blocktrans %}Your account has been locked for security purposes and the system needs to reverify you.{% endblocktrans %}
```

Comment:
I don't think we need to mention account locking here. Perhaps change to:

"The system needs to verify your account."